bosh-upload-release/task

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -ux
+
+# shellcheck disable=SC1091
+source cf-deployment-concourse-tasks/shared-functions
+
+function main() {
+  setup_bosh_env_vars
+
+  TARBALL_VERSION=$(cat version/version)
+
+  TARBALL_TO_UPLOAD="tarballs/${TARBALL_NAME}-${TARBALL_VERSION}.tgz"
```

Comment:
This directory needs to be renamed from `tarballs` to `release-tarball`

Reply:
done